dwave/system/samplers/clique.py

```diff
@@ -0,0 +1,193 @@
+
+            chain_strength (float, optional):
+                The (relative) chain strength to use in the embedding. By
+                default a chain strength of `1.5sqrt(N)` where `N` is the size
+                of the largest clique, as returned by
+                :attr:`.largest_clique_size`.
+
+            **kwargs:
+                Optional keyword arguments for the sampling method, specified
+                per solver in :attr:`.DWaveCliqueSampler.parameters`.
+                D-Wave System Documentation's
+                `solver guide <https://docs.dwavesys.com/docs/latest/doc_solver_ref.html>`_
+                describes the parameters and properties supported on the D-Wave
+                system. Note that `auto_scale` is not supported by this
+                sampler, because it scales the problem as part of the embedding
+                process.
+
+        """
+
+        # some arguments should not be overwritten
+        if 'auto_scale' in kwargs:
+            raise TypeError("sample() got an unexpected keyword argument "
+                            "'auto_scale'")
+        if 'bias_range' in kwargs:
+            raise TypeError("sample() got an unexpected keyword argument "
+                            "'bias_range'")
+        if 'quadratic_range' in kwargs:
+            raise TypeError("sample() got an unexpected keyword argument "
+                            "'quadratic_range'")
+
+        # handle circular import. todo: fix
+        from dwave.system.composites.embedding import FixedEmbeddingComposite
+
+        # get the embedding
+        embedding = find_clique_embedding(bqm.variables, self.target_graph,
+                                          use_cache=True)
+
+        # returns an empty embedding when the BQM is too large
+        if not embedding and bqm.num_variables:
+            raise ValueError("Cannot embed given BQM (size {}), sampler can "
+                             "only handle problems of size {}".format(
+                                len(bqm.variables), self.largest_clique_size))
+
+        assert bqm.num_variables == len(embedding)  # sanity check
+
+        if chain_strength is None:
+            # chain length determines chain strength
+            if embedding:
+                chain_strength = 1.5 * math.sqrt(len(embedding))
+            else:
+                chain_strength = 1  # doesn't matter
+
+            # scale chain strength by the problem scale
+            scalar = max(max(map(abs, bqm.linear.values()), default=0),
+                         max(map(abs, bqm.quadratic.values()), default=0))
+            chain_strength *= scalar
+
+        # # scaling only make sense in Ising space
+        original_bqm = bqm
+
+        if bqm.vartype is not dimod.SPIN:
+            bqm = bqm.change_vartype(dimod.SPIN, inplace=False)
+
+        sampler = FixedEmbeddingComposite(
+            dimod.ScaleComposite(self.child),
+            embedding)
+
+        sampleset = sampler.sample(bqm,
```

Comment:
`kwargs` are not being forward along here!

Reply:
Good catch, fixed